index.html

```diff
@@ -1037,109 +1037,64 @@
 
 </div>
 
-## Principles for Safety, Abuse, and Fraud {#safety}
+## Balancing Conflicting Privacy Rights {#balancing}
 
-There are some cases where privacy and safety are in tension. Many of these cases are
-short-term, only existing until we develop a system that achieves both privacy and
-safety. Long-term cases are rare, but still occasionally come up. We can distinguish
-three categories:
+Sometimes different people's privacy rights come into conflict. Often this happens when
+one user of a service is trying to invade another user's privacy: hiding all private
+information about the first user can prevent the service from protecting the second user.
```

Comment:
Suggestion: "Tension can also arise in cases of fraud and malware, where privacy enables ambiguous or deceptive identity that enables abuse."

Reply:
There was a sense in the [discussion](https://github.com/w3ctag/privacy-principles/blob/main/meetings/2022-01-26-minutes.md#new-topic-105) that I should restrict this section to talking about ways that different privacy interests are in conflict. Otherwise, this document might get too big. Ways that non-privacy principles conflict with our privacy goals should go elsewhere ... although we don't have a good place for them to go.

Reply:
"can prevent" seems like an overstatement. Services can secure data even without successfully collecting all information about a user who tried to invade another's privacy. 

Maybe: Keeping a user's information private may in some cases limit the ways that a service can protect another user.

Reply:
@npdoty We have a couple levels of privacy available here. I wrote "hiding all private information", which is a bit more privacy than your "keeping a user's information private", which is more privacy than "collecting all information about a user". And then those imply a gradient of protection, from my "can prevent" to your "may limit", to more-likely allowing protection.

I think I agree with switching to "may/can limit", since even using all of the first user's plausibly-available information and no specifically-privacy-related restrictions, there's no guarantee that the service will actually be able to protect the second user.